easse/samsa.py

```diff
@@ -229,32 +246,36 @@ def compute_samsa(orig_ucca_passage: Passage, sys_synt_parse):
                 scorea.append(sa)
 
         scoresc = []
-        for i in range(orig_num_scenes):
+        for i in range(num_orig_scenes):
             d = len(scorea[i])
-            v = 0.5*scorem[i] + 0.5*(1/d)*sum(scorea[i])
+            v = 0.5 * scorem[i] + 0.5 * (1 / d) * sum(scorea[i])
             scoresc.append(v)
-        score = (sys_num_sents/(orig_num_scenes**2))*sum(scoresc)
+        score = (num_sys_sents / (num_orig_scenes ** 2)) * sum(scoresc)
     return score
 
 
-def corpus_samsa(orig_sentences: List[str], sys_outputs: List[str], lowercase: bool = False, tokenizer: str = '13a',
+def corpus_samsa(orig_sents: List[str], sys_sents: List[str], lowercase: bool = False, tokenizer: str = '13a',
                  verbose: bool = False):
 
     print('Warning: SAMSA metric is long to compute (120 sentences ~ 1h), disable it if you need fast evaluation.')
```

Comment:
We should update the warning with the new time (I would still keep the warning because 4 minutes / 100 sentences is still quite long compared to other metrics).